### DISCUSSION TITLE
Need help : _TZ3000_hojntt34 (NOUS L13Z) don't be identified like _TZ3000_ruxexjfz (NOUS L13Z)

### DISCUSSION THREAD

**Discussion by scaput69** [Koenkk/zigbee2mqtt#25201] [Devices]
Hello,

i need some help.
I have two 2 gang switch "NOUS L13Z". One is _TZ3000_hojntt34 and the other is _TZ3000_ruxexjfz
The _TZ3000_ruxexjfz is identified good as TS0002 but the _TZ3000_hojntt34 is not good identified like the other. It's identified like SM0001 (only one with contractor).
How can i do to have both identified like TS0002 ?
Thanks for help.

**Comment by Koenkk** [comment]
Added!

Changes will be available in the [dev branch](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html) in a few hours from now. Note that the dev branch currently contains **breaking changes**, see https://github.com/Koenkk/zigbee2mqtt/discussions/24198

**Reply by scaput69** [reply]
Many thanks 😊
